tests/bin/check_tron_jobs_test.py

```diff
@@ -745,6 +745,60 @@ def test_job_running_job_exceeds_expected_runtime(self):
         assert_equal(run['id'], 'MASTER.test.99')
         assert_equal(state, State.STUCK)
 
+    def test_job_waiting_job_exceeds_expected_runtime_already_started(self):
+        job_runs = {
+            'status':
+                'running',
+            'next_run':
+                None,
+            'expected_runtime':
+                480.0,
+            'allow_overlap':
+                True,
+            'runs': [
+                {
+                    'id':
+                        'MASTER.test.100',
+                    'state':
+                        'scheduled',
+                    'run_time':
+                        time.strftime(
+                            '%Y-%m-%d %H:%M:%S',
+                            time.localtime(time.time() + 600),
+                        ),
+                    'end_time':
+                        None,
+                    'start_time':
+                        None,
+                    'duration':
+                        '',
+                },
+                {
+                    'id':
+                        'MASTER.test.99',
+                    'state':
+                        'waiting',
+                    'run_time':
+                        time.strftime(
+                            '%Y-%m-%d %H:%M:%S',
+                            time.localtime(time.time() - 600),
+                        ),
+                    'start_time':
+                        time.strftime(
+                            '%Y-%m-%d %H:%M:%S',
+                            time.localtime(time.time() - 600),
+                        ),
+                    'end_time':
+                        None,
+                    'duration':
+                        '0:10:01.883601',
+                },
+            ],
+        }
```

Comment:
This is a funny indentation style, but it does match the rest of the file.

Reply:
yeah, i think it probably came from black at one point